lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -899,35 +899,113 @@ class MongoClientInterface {
             return cb(errors.InternalError);
         });
     }
-
     /**
-     * Put object when versioning is not enabled
-     * @param {Object} c bucket collection
-     * @param {String} bucketName bucket name
-     * @param {String} objName object name
-     * @param {Object} objVal object metadata
-     * @param {Object} params params
-     * @param {Object} log logger
-     * @param {Function} cb callback
-     * @return {undefined}
+     * Puts an object into a MongoDB collection. Depending on the parameters,
+     * the object is either directly put into the collection
+     * or the existing object is marked as deleted and a new object is inserted.
+     *
+     * @param {Object} collection - The MongoDB collection to put the object into.
+     * @param {string} bucketName - The name of the bucket the object belongs to.
+     * @param {string} objName - The name of the object.
+     * @param {Object} value - The value of the object.
+     * @param {Object} params - Additional parameters. If params.doesNotNeedOpogUpdate is true, the object is directly
```

Comment:
should document the individual parameters:

```suggestion
     * @param {Object} params - Additional parameters
     * @param {bool} params.vFormat - ...
     * @param {bool} params.doesNotNeedOpogUpdate - fs true, the object is directly
```